### PR TITLE
SSL Policy for Certificate is configurable

### PR DESCRIPTION
**Pull request by artusiep**
## what
* Adding new optional variable for SSL Policy
* Adding example with the most secure SSL Policy GCP provide

## why
* configurable SSL Policy should be possible for Atlantis Server to be more secure

### CODE REVIEW

**Review by bschaatsbergen — DISMISSED** [review]
LGTM! 🚀

File: variables.tf [hunk 1]
```diff
@@ -122,6 +122,12 @@ variable "iap" {
   default     = null
 }
 
+variable "ssl_policy" {
+  type        = string
+  description = "The ssl policy name that certificate must follow"
```
Comment (bschaatsbergen) [inline variables.tf:127]: Maybe we can make 'SSL' uppercase here and put a 'the' in front of certificate? :)

**Comment by bschaatsbergen** [comment]
Thanks for raising this PR @artusiep - I've left one cosmetic comment. If you could address this, we can get it merged ASAP.

**Comment by artusiep** [comment]
@bschaatsbergen I have applied your code review suggestions

**Review by bschaatsbergen — APPROVED** [review]
LGTM 🚀

**Comment by bschaatsbergen** [comment]
Thanks for your contribution @artusiep 👏

**Pull request merged**